src/pages/wallets/index.page.tsx

```diff
@@ -20,7 +20,7 @@ import {
   ImTokenIcon,
 } from './icons'
 
-const title = <div>Download a Wallet and Start Exploring the Nervos Ecosystem .</div>
+const title = <div>Download a wallet.</div>
```

Comment:
The title is expected to be `Download a Wallet and Start Exploring the Nervos Ecosystem` in https://docs.google.com/document/d/1abU8b1RSSDIXN3IquKbdT7ukxLCoIyaZg8F32-XtJow/edit

Reply:
This is the designer's comment https://nervos-official-website-git-preview-magickbase.vercel.app/wallets#thread-id=_pVf5